tools/harness-thci/OpenThread.py

```diff
@@ -254,7 +254,7 @@ def __getIp6Address(self, addressType):
                 # link local address
                 if ip6Addr.split(':')[4] != '0':
                     linkLocal64Addr = ip6Addr
-            elif ip6AddrPrefix == 'fd00':
+            elif ip6Addr.startswith(self.meshLocalPrefix):
```

Comment:
use self.localprefix directly instead of defining self.meshLocalPrefix?

Reply:
Ok. But `localprefix` doesn't obey the camel naming rule. So I changed them into `meshLocalPrefix`.